### PR TITLE
Linking may take time - Info Content added

### PR DESCRIPTION
**Pull request by mounikahmcts**
https://tools.hmcts.net/jira/browse/DIV-4419

It's taking a while for users IDAM accounts to be associated with the relevant CCD case - up to two minutes. Below content added
"It may take a few minutes to finish setting up your account. Please don't leave this page or refresh it."

### CODE REVIEW

File: steps/capture-case-and-pin/CaptureCaseAndPin.content.json [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
+    "referenceNumber": "Reference number",
     "referenceNumberLabel": "This is a 16-digit number",
     "securityAccessCode": "Access code",
     "securityAccessCodeLabel": "Your code is 8 characters long",
+    "infoAboutSetup":"It may take a few minutes to link your account with this case. Please don't leave this page or refresh it.",
     "thereWasAProblem": "There was a problem",
-    "referenceNumberOrPinDoNotMatch": "This case number and/or access code are incorrect. Try entering these details again. Contact the divorce centre if you need help.",
-    "errorLinkingCase": "This case number and/or access code are incorrect. Try entering these details again. Contact the divorce centre if you need help.",
+    "referenceNumberOrPinDoNotMatch": "The reference number or access code is incorrect. Please try entering these details again. You can contact the Courts and Tribunals Service Centre if you need help.",
+    "errorLinkingCase": "The reference number or access code is incorrect. Please try entering these details again. You can contact the Courts and Tribunals Service Centre if you need help.",
```
Comment (johnoneill46) [inline steps/capture-case-and-pin/CaptureCaseAndPin.content.json]: Lgtm - just wondering why there is duplicated content though, could we not just use the one line?

Comment (alexwakeman) [reply steps/capture-case-and-pin/CaptureCaseAndPin.content.json]: @johnoneill46 - this was a result of a backend issue - we return a 404 no matter what and needed this message for that and for a 401. Can't remember exact details but this was needed to cover both these scenarios - not ideal admittedly.

Comment (johnoneill46) [reply steps/capture-case-and-pin/CaptureCaseAndPin.content.json]: Certainly seems a but strange but no problem!

**Comment by nsingamsetty** [comment]
This is tested and merging on behalf of Phi.please refer the ticket DIv-4419 for more details

**Pull request merged**